misc/python/materialize/checks/scenarios.py

```diff
@@ -107,17 +110,17 @@ def actions(self) -> List[Action]:
         return [
             StartMz(),
             StartClusterdCompute(),
-            UseClusterdCompute(base_version=self.version_cargo),
-            Initialize(self.checks(), base_version=self.version_cargo),
+            UseClusterdCompute(base_version=self.base_version()),
```

Comment:
UseClusterdCompute can also use a constructor that takes a `Scenario` as its only argument.

However, keep the `StartMz` and `StartClusterdCompute` constructors as they are so that it is more obvious which Mz version is being started by them.